elastic/kibana
# [RAM] Make the Cancel Snooze button skip the current recurrence

## Summary

Part of #132860. This updates the backend to allow the Cancel Snooze button to skip the currently active recurrence of a snooze, while keeping future recurrences of that snooze schedule intact.

This does NOT incorporate the UI changes in #132860. These will merge in a later PR.

**To test**

- Schedule a snooze that starts NOW and recurs every day/week/etc
- Refresh the page, press the Cancel Snooze button
- Ensure that the current snooze is cancelled but the snooze schedule still exists

### Checklist
